ClinGen/clincoded
# Connects to #1301. Connects to #1375. Connects to #1378. Connects to #1391.

**Steps to test #1301:**
1. Create a new **Group** curation in the GCI.
2. Fill out the **required** fields. **Associate** a disease with this Group evidence and add `HP:0010704, HP:0030300` to the **Phenotype(s) in Common (HPO ID(s))** text field.
3. Click the **Save** button and click on the **Add New Family for this Group** button to proceed to the **Family** evidence curation.
4. In the **Family** curation, click the **Copy all Phenotype(s) from Associated Group** button and expect to see the `HP:0010704, HP:0030300` string being added to the Family's **Phenotype(s) in Common (HPO ID(s))** text field.
5. Without adding a Proband to the Family, click the **Save** button. Expect to be able to save the curation successfully without being prompted for adding a Proband and its associated disease.

**Steps to test #1375:**
1. Create a new **Family** curation in the GCI.
2. Fill out the **required** fields. DO NOT associate a disease with this Family evidence.
3. Add a Proband and associate a disease with it (as it is **required**), and add a variant to the Proband (optionally). You may try a disease that is either in the test data (e.g. **DOID:13636**, **OMIM:601200**) or not (e.g. **DOID:0060233**, **OMIM:311250**).
4. Click the **Save** button. On the subsequent page, click on the **Score/Add information about proband** button to proceed to the **Proband Individual** evidence curation if you added a variant to the Proband. Otherwise, select the **No** option for the **No segregating variant information has been associated with this Family. Would you like to add it?** selection to proceed to the **Proband Individual** evidence curation.
5. In the Proband **Individual** curation, expect to see the associated disease (entered in the **Family** curation) being shown.
6. Click on the red delete disease button and expect to see the **Copy disease term from Gene-Disease Record** button being shown (because this individual is not associated with a Group and its associated Family has no associated disease).
![image](https://user-images.githubusercontent.com/6956310/28838334-ecb861fe-76a4-11e7-8273-b7922072e4dc.png)

**Steps to test #1378:**
1. Create a new **Individual** curation in the GCI.
2. Fill out the **required** fields. Select the **Yes** option for the **Is this Individual a proband** selection and expect to see the **Score Proband** section panel being shown.
3. Add a variant and expect to see the **Select Status** selection becoming active in the **Score Proband** section panel.
4. Select the **Score** option from the **Select Status** selection and leave the **Confirm Case Information type** selection at the **No Selection** option.
5. Click on the **Save** button and expect to see a warning message in the yellow box being shown.
![image](https://user-images.githubusercontent.com/6956310/28837419-84240cfe-76a1-11e7-8097-85d70961124a.png)
6. Now select an option (other than **No Selection**) for the **Confirm Case Information type** selection. Expect to see the warning message no longer being shown.

**Steps to test #1391:**
1. Go to VCI at `^/select-variant/` and add a **ClinVar ID** (e.g. 139214).
2. Upon landing at the **Evidence View** at `^/variant-central/?variant=[uuid]`, click on the **Predictors** tab.
3. Under the **Missense** sub-tab, click on the **Analyze using HumanSplicingFinder** external link.
4. Expect to see a new tab/window being opened with the **Human Splicing Finder** landing page at `http://www.umd.be/HSF3/HSF.shtml`.
5. Now click on the **Silent & Intron** sub-tab and click on the **Analyze using HumanSplicingFinder** external link. Expect to see the same behavior as described in Step 4.
